src/MoveUntilTouchController.cpp

```diff
@@ -187,6 +187,8 @@ void MoveUntilTouchController::setForceTorqueThreshold(FTThresholdGoalHandle gh)
     result.message =
         "Must wait until initial taring of force/torque sensor is complete "
         "before setting thresholds or sending trajectories.";
+    gh.setAccepted();
```

Comment:
It seems like since `result.success` is false, this will actually be set to `REJECTED` below.

Reply:
Oh, yes, I intended to return here.